starpicker/reviews.py

```diff
@@ -77,7 +77,10 @@ def send_to_slack(self):
         }
 
         for webhook_url in config.SLACK_WEBHOOK_URLS:
-            requests.post(webhook_url, json=body)
+            r = requests.post(webhook_url, json=body)
```

Comment:
Might as well call it `response` in the actual code, don't you think? Especially since we already have an `R`.